res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -62,6 +56,13 @@ var intervalsPerRev = 1200,
     rpm = 33 + 1 / 3,  //Like a real vinyl !!! :)
     alpha = 1.0 / 8,   //Adjust to suit.
     beta = alpha / 32; //Adjust to suit.
+    
+/**************************
+ * Loop Size array 
+ * first 4 values used for Autoloop Not shifted
+ * last 4 values used for Autoloop Shifted
+ **************************/    
+var loopsize = [1, 2, 4, 8, 0.125, 0.25, 0.5, 16];
```

Comment:
It may make more sense to break from their mappings for this. What do you think of 2, 4, 8, 16 beats unshifted and 1, 1/2, 1/4, 1/8 beats shifted?